packages/twenty-server/src/engine/api/graphql/workspace-schema-builder/utils/generate-fields.utils.ts

```diff
@@ -58,7 +58,9 @@ export const generateFields = <
       ? {
           nullable: fieldMetadata.isNullable,
           defaultValue: fieldMetadata.defaultValue,
-          isArray: fieldMetadata.type === FieldMetadataType.MULTI_SELECT,
+          isArray:
+            kind !== InputTypeDefinitionKind.Filter &&
```

Comment:
I don't get this logic but it was already misleading, looks like a patch on something not correct!